.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "node-tap.testFilePatterns": [
+    "test/*",
+    "*.test.js"
+  ]
+}
```

Comment:
can you please add this folder to .gitignore?

Reply:
Sorry missed that, done